astropy/coordinates/baseframe.py

```diff
@@ -1330,10 +1330,12 @@ def __setattr__(self, name, value):
         else:
             super(GenericFrame, self).__setattr__(name, value)
 
-# Note: this is a base class that defines default representation info for frames
-# that represent longitude and latitude with RA, Dec
+
 class BaseRADecFrame(BaseCoordinateFrame):
     """
+    A base class that defines default representation info for frames that
```

Comment:
Ah, I explicitly didn't add this to the docstring because I use this docstring in the subclass docstrings, e.g., [here](https://github.com/astropy/astropy/pull/6219/files#diff-4148cf6ed0329aa00fd7c25933688986R28). We could keep it, and just strip it from the base docstrings?